### DISCUSSION TITLE
Duplicate

### DISCUSSION THREAD

**Discussion by BarryYuniwo** [community/community#123447] [Programming Help] [answered]
### Select Topic Area

Question

### Body

How do i remove duplicate from an array

**Comment by Sackitey68** [comment, marked as answer]
// Given this is the array

const arr = [1, 2, 3, 1, 2, 5, 1]
const duplicateRemoved = [...new set(arr)]
console.log(duplicateRemoved)

// this will removed the duplicate

**Reply by ebndev** [reply]
@Sackitey68 thanks for taking the time to answer 😄
